substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -879,6 +914,22 @@ private ResolvedJavaMethod promoteAnalysisToHosted(ResolvedJavaMethod m) {
             return m;
         }
 
+        private ResolvedJavaMethod originalMethod() {
+            // unwrap to an original method as far as we can
+            ResolvedJavaMethod targetMethod = method;
+            while (targetMethod instanceof WrappedJavaMethod) {
+                targetMethod = ((WrappedJavaMethod) targetMethod).getWrapped();
+            }
+            // if we hit these two substitutions then we can translate to the original
+            // for identity otherwise we use whatever we unwrapped to.
+            if (targetMethod instanceof SubstitutionMethod) {
+                targetMethod = ((SubstitutionMethod) targetMethod).getOriginal();
+                // } else if (targetMethod instanceof CustomSubstitutionMethod) {
+                // targetMethod = ((CustomSubstitutionMethod) targetMethod).getOriginal();
```

Comment:
Are these commented out by mistake?

Reply:
This was deliberate and the comments have now been removed.